python/test_requirements.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+pytest
```

Comment:
I think this can be defined in the `setup.cfg` as well via the `extra_requires`?

Reply:
Done